src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.cpp

```diff
@@ -253,6 +253,11 @@ CPUTestsBase::CPUInfo CPUTestsBase::getCPUInfo() const {
     return makeCPUInfo(inFmts, outFmts, priority);
 }
 
+#if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
```

Comment:
It looks like we should use OV_CPU_WITH_ACL definition instead

Reply:
Updated